maplibre_gl/lib/src/controller.dart

```diff
@@ -304,6 +304,9 @@ class MapLibreMapController extends ChangeNotifier {
 
   final MapLibrePlatform _maplibrePlatform; //ignore: unused_field
 
+  // Tracks whether the controller has already been disposed
+  bool isDisposed = false;
+
```

Comment:
nit: Can you make _isDisposed private and only expose a getter so that this property is not changeable from outside?